src/core/Core.js

```diff
@@ -415,7 +415,7 @@ class Uppy {
       throttledCalculateProgress(data)
     })
 
-    this.on('core:upload-success', (fileID, uploadResp, uploadURL) => {
```

Comment:
I think someone specifically asked for a full response object, and it makes sense. Is this removing uploadResp, or am I missing something?